go.mod

```diff
@@ -31,3 +31,5 @@ require (
 	golang.org/x/text v0.10.0 // indirect
 	google.golang.org/protobuf v1.30.0 // indirect
 )
+
+replace github.com/mdlayher/ndp => github.com/jmbaur/ndp v0.0.0-20240310235550-d810a7ebc0fa
```

Comment:
Depends on these changes. Plan is to remove this if/when the ndp changes are merged

Reply:
SGTM! I will happily accept this contribution and the one to ndp.